vm/src/circuit/riscv.rs

```diff
@@ -454,24 +473,37 @@ fn parse_J(cs: &mut R1CS, J: u32) {
     cs.mul("J=19", "opcode=19", "f3=0"); // addi
     cs.set_var("J=20", 0); // subi does not exist
     cs.mul("J=21", "opcode=19", "f3=1"); // slli
+    cs.nand("J=21", "f7"); // f7 == 0 if slli
```

Comment:
Similarly, we should document that this isn't actually `f7`, it's `imm[11:6]`, which coincides with `f7`.

Reply:
I don't see it as having `funct7` per the spec:

<img width="591" alt="Screenshot 2024-07-11 at 11 10 52" src="https://github.com/nexus-xyz/nexus-zkvm/assets/20324824/aa1bc82c-a1c4-458c-9ab8-b32f784c019d">

Am I looking at the wrong version?

Reply:
I'm seeing the table. Your comment just made me realize that those bits are now not called funct7.

Reply:
I rephrased the comments.